example/cypress/support/commands.js

```diff
@@ -24,4 +24,4 @@
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
-import 'cypress-file-upload';
+import '../../../index.js';
```

Comment:
I understand that by this you're not changing anything but importing `"../blabla/index.js"` may be unclear (or at least not so transparent) in comparison to seeing a package name import.
Right now it's just the dumber version, i.e. `"cypress-file-upload"` in `devDependencies` list, explicit import in `commands.js` – so no chance to make it somewhat confusing.

Reply:
difference:
- `import "cypress-file-upload"` test only last version on npm
- `import "../blabla/index.js"` test real code from project

with out it edits, example/cypress not using my changes from PR.

Reply:
I understand that too.
Let's maybe make it this way:
```
import "../blabla/index.js"
// -- In your application the import above should be replaced with: --
// import "cypress-file-upload"
```

Reply:
@abramenal move to #33